application/src/main/java/io/redlink/smarti/webservice/RocketChatEndpoint.java

```diff
@@ -213,6 +210,35 @@ private String toJsonString(SmartiUpdatePing smartiUpdatePing) throws JsonProces
         }
     }
 
+    /**
+     * Allow conversation-independent search.
+     * @param clientName the client id
+     * @param queryParams the actual query-params
+     */
+    @ApiOperation(value = "search for a conversation", response = SearchResult.class,
+            notes = "besides simple text-queries, you can pass in arbitrary solr query parameter.")
+    @RequestMapping(value = "{clientId}/search", method = RequestMethod.GET)
```

Comment:
As far as I understand: You're extending the rocket-chat service endpoint for a `{clientId}/search` function/call. So it is ensured that I will only get results from that `{clientId}`? Is that right?

Reply:
:+1: